migrations/V209__Update_MTO_Common_Solution_Contact.sql

```diff
@@ -85,3 +119,11 @@ EXECUTE FUNCTION ENFORCE_SINGLE_PRIMARY_CONTACT();
 
 -- This needs to be part of a separate migration than the definition of contractor table to avoid issues with existing data 
 ALTER TYPE TABLE_NAME ADD VALUE 'mto_common_solution_contractor';
+
+CREATE UNIQUE INDEX IF NOT EXISTS uniq_user_id_per_solution_key
+ON mto_common_solution_contact (mto_common_solution_key, user_id)
+WHERE user_id IS NOT NULL;
+
+CREATE UNIQUE INDEX IF NOT EXISTS uniq_mailbox_address_per_solution_key
+ON mto_common_solution_contact (mto_common_solution_key, mailbox_address)
```

Comment:
I think this works well! The email I think is the primary piece we should be concerned about. I was wondering if we wanted to enforce it on the mailbox name as well, but I think this is ok as is.